lib/tpm2_session.h

```diff
@@ -146,6 +148,37 @@ TPMI_SH_AUTH_SESSION tpm2_session_get_session_handle(tpm2_session *session);
 tpm2_session *tpm2_session_new(TSS2_SYS_CONTEXT *sapi_context,
         tpm2_session_data *data);
 
+/**
+ * Saves session data to disk allowing tpm2_session_from_file() to
+ * restore the session.
+ *
+ * @Note
+ * This is accomplished by calling:
+ *   - Tss2_Sys_ContextSave - marks to some RMs like abrmd not to flush this session
```

Comment:
I would say tpm2-abrmd instead of just abrmd.

Reply:
roger